divide_and_conquer/fibonacci_sequence_dnc.py

```diff
@@ -0,0 +1,63 @@
+"""
+We can find the Fibonacci sequence more effectively than naive bottom-up algorithm
+    We use One of the well-known properties of Fibonacci number is:
+
+    | F(n+1) ,  F(n)   |       | 1 , 1 | ^ n
+    |                  |   =   |       |
+    | F(n)   ,  F(n-1) |       | 1 , 0 |
+
+    We can therefore compute the (n)th power of A = ((1,1),(1,0)) using recursive squaring
+
+    Time complexity: O(log(n)), because it is a divide and conquer algorithm
+"""
+
+
+def get_fibonacci(n):
+    """
+    :param n: number of Sequence
+    :return: (n)th sequence of Fibonacci sequence
+
+    >>> get_fibonacci(5)
+    5
+    >>> get_fibonacci(10)
+    55
+    >>> get_fibonacci(20)
+    6765
```

Comment:
Write a **doctest** for a case where `n < 2`, also what about `n < 1` and `n < 0`